### PR TITLE
Adding book model

### PR DESCRIPTION
**Pull request by JoelIvanStella**
## Summary
- Adding book model with persistence annotations
## Trello Card
https://trello.com/c/Ds56gPvD
## Test
image

### CODE REVIEW

File: src/main/java/com/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = true)
```
Comment (torsello) [inline src/main/java/com/wolox/training/models/Book.java:18]: Hi @JoelIvanStella. If you're only going to define the non-nullability, don't use @column but instead use @NotNull (spring-boot-starter-validation). Change it for every case :+1: 

If variable is nullable, you can just leave it without annotations.

Also, leave a blank line between variables.

**Review by torsello — CHANGES_REQUESTED** [review]

File: src/main/java/com/wolox/training/models/Book.java (hunk 1 above)
Comment (jdhiguita) [reply src/main/java/com/wolox/training/models/Book.java:18]: The @column annotations are required to be able to generate the model in the database.
In this case I would add the @column (nulleable = false) annotation in conjunction with the @NotNull annotation(this should in the rest of the fields except for the id)

Comment (JoelIvanStella) [reply src/main/java/com/wolox/training/models/Book.java:18]: Thanks guys! I pushed the change.

**Review by torsello — APPROVED** [review]
Good Job!

**Review by jdhiguita — APPROVED** [review]
Great job!

**Pull request merged**